gemhardware/include/gem/hw/HwGenericAMC.h

```diff
@@ -61,112 +61,124 @@ namespace gem {
-        //return N_GTX; }
 
         /**
-         * Check if the firmware supports GBT communication
-         * @returns whether or not the firmware supports GBT communication
+         * FIXME: OBSOLETE
+         * Check if the AMC FW supports GBT communication
+         * @returns whether or not the AMC FW supports GBT communication
          */
         uint32_t supportsGBTLink() {
           return readReg(getDeviceBaseNode(),"GEM_SYSTEM.CONFIG.USE_GBT"); }
 
         /**
-         * Check if the firmware supports trigger links
-         * @returns whether or not the firmware supports trigger links
+         * Check if the AMC FW supports trigger links
+         * @returns whether or not the AMC FW supports trigger links
          */
         uint32_t supportsTriggerLink() {
           return readReg(getDeviceBaseNode(),"GEM_SYSTEM.CONFIG.USE_TRIG_LINKS"); }
 
         /**
-         * Read the system firmware register
-         * @param system determines whether to read the system or user firmware register
-         * @returns a string corresponding to firmware version
+         * Read the AMC FW register
+         * @param system determines whether to read the system (default) or user FW register
+         * @returns a string corresponding to AMC FW version
          */
         virtual std::string getFirmwareVer(bool const& system=true);
 
         /**
-         * Read the system firmware register
-         * @param system determines whether to read the system or user firmware register
-         * @returns the firmware version as a 32 bit unsigned
+         * Read the AMC FW register
+         * @param system determines whether to read the system (default) or user FW register
+         * @returns the AMC FW version as a 32 bit unsigned
          */
         virtual uint32_t getFirmwareVerRaw(bool const& system=true);
 
         /**
-         * Read the system firmware register
+         * Read the AMC FW register
+         * @param system determines whether to read the system (default) or user FW register
          * @returns a string corresponding to the build date dd-mm-yyyy
          */
         virtual std::string getFirmwareDate(bool const& system=true);
 
         /**
-         * Read the system firmware register
-         * @param system determines whether to read the system or user firmware register
+         * Read the AMC FW register
+         * @param system determines whether to read the system (default) or user FW register
          * @returns the build date as a 32 bit unsigned
          */
         virtual uint32_t getFirmwareDateRaw(bool const& system=true);
 
         //user core functionality
         /**
-         * Read the user firmware register
-         * @returns a hex number corresponding to the build date
-         * OBSOLETE in V2 firmware
+         * Read the user AMC FW register
+         * @returns the user FW build date as a 32-bit unsigned value
+         * OBSOLETE in V2 AMC FW
```

Comment:
I suggest `@note` or `@warning` here, or even `[[deprecated]]` if V1 firmware is going to be phased out.

Reply:
Yeah, these weren't `doxygen` comments, rather just notes for me to clean up later :-D